tensorflow/compiler/xla/debug_options_flags.cc

```diff
@@ -42,9 +42,8 @@ DebugOptions DefaultDebugOptionsIgnoringFlags() {
   opts.set_xla_cpu_use_mkl_dnn(true);
 #endif  // INTEL_MKL
   opts.set_xla_gpu_max_kernel_unroll_factor(4);
-  // Set cudnn batchnorm off by default; it does not provide a performance win
-  // on average.
-  opts.set_xla_gpu_use_cudnn_batchnorm(false);
+  // Set cudnn batchnorm on by default.
+  opts.set_xla_gpu_use_cudnn_batchnorm(true);
```

Comment:
Hi, do you mind splitting out this part of the change?  That way we can land this bit without any performance implications and the test the performance on our internal benchmarks without risking a regression.

Reply:
You can probably set this explicitly in the unit tests so that we know we're testing the right thing.

Reply:
Sure I can split this. Actually the tests always explicitly runs both the cudnn and batchnorm-expanded+fusion version irrespective of what the default is set here. I had the same concern but then I verified this.  The test sets `use_cudnn_batchnorm` parameter during construction to `false` and then flips it . See https://github.com/tensorflow/tensorflow/blob/07b5ca443887fd6d739ef6055e6cd3f2c02ba226/tensorflow/compiler/xla/tests/batch_normalization_test.cc#L95 (`class BatchNormalizationTest`). Similar thing happens for the other `class BatchNormTestManySizes` (https://github.com/tensorflow/tensorflow/blob/07b5ca443887fd6d739ef6055e6cd3f2c02ba226/tensorflow/compiler/xla/tests/batch_normalization_test.cc#L405)